### DISCUSSION TITLE
show result processing in screen

### DISCUSSION THREAD

**Discussion by ai-samira** [orchidsoftware/platform#2598] [Q&A] [answered]
![ping](https://user-images.githubusercontent.com/118598363/235622176-715e412b-981e-479b-9671-4322dc68143d.png)

I want to have a screen that takes a value from the user and processes the value by pressing the button and displays the result to the user.
Now I get the value and do the processing but how can I show the result on the screen (like textaria) to the user.
Thanks

**Comment by tabuna** [comment, marked as answer]
You can write the result to be shown in a database, session or simply pass in the GET parameters of the URL address.

**Reply by ai-samira** [reply]
Thanks, It's done
